packages/web/src/components/tipping/tip-audio/SendTip.tsx

```diff
@@ -98,12 +94,21 @@ export const SendTip = () => {
   useEffect(() => {
     if (!profile) return
 
-    // todo: make sure these are already pre-sorted by highest support amount
-    const newTopSupporter = supportersMap[profile.user_id]?.length
-      ? supportersMap[profile.user_id][0]
+    const supportersForProfile = supportersMap[profile.user_id] ?? {}
```

Comment:
not sure if ive seen other similar code paths in other comps, maybe a chance to share code?